lib/bundler/settings.rb

```diff
@@ -256,5 +280,11 @@ def normalize_uri(uri)
       uri
     end
 
+    def print_remembered_flags_deprecation_warning
+      Bundler.ui.warn "Starting in Bundler 2.0, flags passed into commands will no longer be remember past that command." \
+        " Instead please set flags you want remembered between commands using `bundle config <setting name> <setting value>`" \
+        " To ignore this warning use `bundle config ignore_warnings.remembered_flags true`"
+      set_current "ignore_warnings.remembered_flags", true
```

Comment:
Yeah, we should warn in 1-99 and just not remember flags in 2-0. I don't think @pducks32 knew that 1-99 existed when he wrote this PR, though...